src/assets/icons/edit-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please check with these Icons, that we dont create duplicate ones. We should have an edit, or a trash icon already.

If you import them as react components with the `?react` in the end, you can apply regular tailwind to it, including `fill` and dimensions.